org.jacoco.core.test/src/org/jacoco/core/internal/data/CRC64Test.java

```diff
@@ -27,6 +28,9 @@
 
 	@Test
 	public void testJava9() {
+		assertEquals(0x589E9080A572741EL,
```

Comment:
Separate test per Java version?

Reply:
@marchof I would prefer to keep single test, so that values are written next to each other to make clear that they are all different except for Java 9 that is equal to Java 8. So just renamed test.